Userland/Libraries/LibWeb/DOM/AccessibilityTreeNode.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2022, Jonah Shafran <jonahshafran@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/JsonObjectSerializer.h>
+#include <AK/Vector.h>
+#include <LibWeb/DOM/Node.h>
+#include <LibWeb/Forward.h>
+
+namespace Web::DOM {
+
+struct AccessibilityTreeNode {
+    DOM::Node const* value { nullptr };
+    Vector<AccessibilityTreeNode> children;
```

Comment:
I had originally thought it was safe do so since the lifetime of the tree was limited to when it is requested. However, on reflection, this doesn't make sense and exposes us to potential issues with the DOM tree getting modified after we build the tree but before we serialize. I've gone ahead and made this GC-allocated.